### PR TITLE
[24.1] New launch form

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Documents the new launch form. Port from https://github.com/seqeralabs/docs/pull/24/files

### CODE REVIEW

File: platform_versioned_docs/version-24.1.0/launch/advanced.mdx [hunk 1]
```diff
@@ -39,26 +38,38 @@ reports:
     mimeType: "text/html"
 ```
 
-## Pre and post-run scripts
+### Pre and post-run scripts
+
+Run custom code either before or after the execution of the Nextflow script. These fields allow you to enter shell commands.
+
+**Pre-run scripts** are executed in the nf-launch script prior to invoking Nextflow processes. Pre-run scripts are useful for executor setup (such as loading a private CA certificate) and troubleshooting. For example, add `sleep 3600` to your pre-run script to instruct Nextflow to wait 3600 seconds before process execution after the nf-launcher container is started, to create a window in which to test connectivity and other issues before your Nextflow processes execute.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1.0/launch/advanced.mdx:45]:
```suggestion
**Pre-run scripts** are executed in the nf-launch script prior to invoking Nextflow processes. Pre-run scripts are useful for executor setup (such as loading a private CA certificate) and troubleshooting. For example, add `sleep 3600` to your pre-run script to instruct Nextflow to wait 3600 seconds before process execution after the nf-launcher container is started. This allows you to create a window in which to test connectivity and other issues before your Nextflow processes execute.
```

File: platform_versioned_docs/version-24.1.0/launch/advanced.mdx [hunk 2]
```diff
@@ -39,26 +38,38 @@ reports:
...
 
-You can run custom code either before or after the execution of the Nextflow script. These fields allow you to enter shell commands.
+**Post-run scripts** are executed after all Nextflow processes have completed. Post-run scripts are useful for triggering a third party service via an API request.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1.0/launch/advanced.mdx:47]:
```suggestion
**Post-run scripts** are executed after all Nextflow processes have completed. One use for for post-run scripts is to trigger a third-party service via an API request.
```

**Review by justinegeffen — COMMENTED** [review]
Two comments but otherwise it's looking good.

File: platform_versioned_docs/version-24.1.0/launch/launchpad.mdx [hunk 3]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
-:::
+- **Labels**: Assign new or existing [labels](../labels/overview.mdx) to the run.
+
+- **Compute environment**: The [compute environment](../compute-envs/overview.mdx) where the run will be launched.
+
+- **Work directory**: The cloud storage or file system path where pipeline scratch data is stored. Seqera will create a scratch sub-folder if only a cloud bucket location is specified. Use file system paths for local or HPC compute environments.
 
-**Quick launch an unconfigured pipeline**
+  :::note
+  The credentials associated with the compute environment must have access to the work directory if it resides in cloud storage.
```
Comment (mbosio85) [inline platform_versioned_docs/version-24.1.0/launch/launchpad.mdx:49]: I believe this holds true for all storages

File: platform_versioned_docs/version-24.1.0/launch/launchpad.mdx [hunk 4]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
-    :::note
-    Nextflow pipelines are Git repositories that can reside on any public or private Git-hosting platform. See [Git integration](../git/overview.mdx) in the Seqera docs and [Pipeline sharing](https://www.nextflow.io/docs/latest/sharing.html) in the Nextflow docs for more details.
-    :::
+There are three ways to enter **Run parameters** prior to launch:
+
+- The **Input form view** displays form fields to enter text, select attributes from dropdowns, and browse input and output locations with [Data Explorer](../data/data-explorer.mdx).
+- The **Config view** displays a raw schema that you can edit directly. Select JSON or YAML format from the **View as** dropdown.
+- **Upload params file** allows you to upload a JSON or YAML file with run parameters.
+
+Seqera uses a `nextflow_schema.json` file in the root of the pipeline repository to dynamically create a form with the necessary pipeline parameters. All pipelines contain at least input and output parameters:
```
Comment (mbosio85) [inline platform_versioned_docs/version-24.1.0/launch/launchpad.mdx:60]: is this true? do all pipeline with a schema have input and output parameters or is it an nf-core convention that can be skipped?

File: platform_versioned_docs/version-24.1.0/launch/launchpad.mdx (hunk 3 above)
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/launch/launchpad.mdx:49]:
```suggestion
  The credentials associated with the compute environment must have access to the work directory.
```

File: platform_versioned_docs/version-24.1.0/launch/launchpad.mdx [hunk 5]
```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
...
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/launch/launchpad.mdx:60]:
```suggestion
Seqera uses a `nextflow_schema.json` file in the root of the pipeline repository to dynamically create a form with the necessary pipeline parameters. Most pipelines contain at least input and output parameters:
```

**Review by mbosio85 — APPROVED** [review]
all good for me :)

**Pull request merged**